### DISCUSSION TITLE
onebot中的endpoint如何填写

### DISCUSSION THREAD

**Discussion by stars-spark** [koishijs/koishi#940] [Q&A] [answered]
另外使用echo时，出现了权限不足的错误

**Comment by HuanLinOTO** [comment, marked as answer]
你好
首先感谢你对 Koishi 项目组解决问题的能力的认可
其次 如果问简单的问题 请去 QQ 群 426230045 提问, 使用 issue 提问开发问题是十分难蚌的

再其次 送您几张图看看
![K~UH}{)Q0IP 6SXMP5QQDD0](https://user-images.githubusercontent.com/94176676/213106329-980fc58a-7532-434a-ae9d-d67f75791822.png)
![}A_S 7RTLHY5NHL~`%Q@_X7](https://user-images.githubusercontent.com/94176676/213106337-4f03c759-388e-41d8-bc1f-14c25c87dfc2.png)
![Q8 N3DGL_E2%}M`6BQN557R](https://user-images.githubusercontent.com/94176676/213106343-51712b50-b1b7-465f-a9f4-a425fa70852b.jpg)

**Comment by neveler** [comment]
> 另外使用echo时，出现了权限不足的错误

issue 一般是用来反馈漏洞的，求助可以去discuss或者qq群 426230045。

**Comment by MaikoTan** [comment]
It is okay to ask question here, everybody should answer the question friendly.

### Re: configurations of onebot endpoint

First of, I assume you have read the [`onebot and gocqhttp configuration`](https://koishi.chat/plugins/adapter/onebot.html#go-cqhttp-%E9%85%8D%E7%BD%AE%E5%8F%82%E8%80%83) section of the koishi document. The `endpoint` was depends on the communication method of the go-cqhttp that you chosen. So unless you have posted the configuration of yours, we cannot provide any advise for that.

### Re: `echo` command previledge

As the `echo` command requires at least level `2` authority to invoke, so you cannot use the command until you have set your authority as `2` or higher.
A simple approach for this is `dataview` plugin, which is a tab page of the Koishi Console, you could find your user name in the `user` table, and then set your `authority` as `2` or higher.

------

在这里提问是可以的，大家要友善的回答问题。

### 回复：onebot 端点的配置

首先，我假设您已经阅读了 [`onebot 和 gocqhttp 配置`](https://koishi.chat/plugins/adapter/onebot.html#go-cqhttp-%E9%85%8D%E7%BD%AE%E5%8F%82%E8%80%83) 恋石文档的部分。 `endpoint` 取决于您选择的 go-cqhttp 的通信方法。 因此，除非您发布了您的配置，否则我们无法提供任何建议。

### 回复：`echo` 命令优先权

由于 `echo` 命令至少需要 2 级权限才能调用，因此您必须将权限设置为 2 级或更高才能使用该命令。
一个简单的方法是 `dataview` 插件，它是恋石控制台的一个标签页，您可以在 `user` 表中找到您的用户名，然后将您的 `authority` 设置为 `2` 或更高。

**Reply by stars-spark** [reply]
Thank you for your kind answer.🌹🌹🌹

**Comment by MaikoTan** [comment]
Update: the `echo` command documentations are [here](https://koishi.chat/plugins/common/echo.html#%E6%8C%87%E4%BB%A4-echo)

更新：`回声`的指令的文档在[这里](https://koishi.chat/plugins/common/echo.html#%E6%8C%87%E4%BB%A4-echo)

**Comment by HuanLinOTO** [comment]
本土化翻译:

在这里提问是可以的，大家要友善的回答问题。

回复：onebot endpoint 的配置
首先，我假设您已经阅读了 [onebot 和 gocqhttp 配置](https://koishi.chat/plugins/adapter/onebot.html#go-cqhttp-%E9%85%8D%E7%BD%AE%E5%8F%82%E8%80%83) Koishi文档的部分。 endpoint 取决于您选择的 go-cqhttp 的通信方法。 因此，除非您 ws-reserve 的方式让 Koishi 自启动 gocqhttp，要不然我们无法提供任何建议。

回复：echo 命令权限问题
由于 echo 命令至少需要 2 级权限才能调用，因此您必须将权限设置为 2 级或更高才能使用该命令。
一个简单的方法是 dataview 插件，它 Koishi 控制台的一个左侧边栏的标签，您可以在 user 表中找到您的用户名，然后将您的 authority 设置为 2 或更高。

**Comment by ilharp** [comment]
楼上 @MaikoTan 的回复是错误的，issue 板块只用来反馈 Koishi 的 bug，或提出功能建议（FR）；不能用于解答使用上的问题。

鉴于本 issue 的 author 已经在 24 小时内打开了过多 issue，现在我将直接将所有 3 个 issue 转换为 discussion 后锁定。如需继续交流，请前往对应的 discussion。

也请 author 和 team member 都注意一下 Koishi 官方文档中的 [参与讨论](https://koishi.chat/about/contact.html) 页面，可以通过 QQ 群、Discord、论坛和 discussion 进行提问，不要使用 issue。后续遇到此类问题麻烦在回答之前先 lock 并转换至 discussion。

**Reply by MaikoTan** [reply]
Okay, forgot it, will do next time, sorry.

**Reply by stars-spark** [reply]
我很抱歉
